app/models/vm_or_template.rb

```diff
@@ -1061,6 +1061,10 @@ def miq_server_proxies
     when 'VMware'
       # VM cannot be scanned by server if they are on a repository
       return [] if storage_id.blank? || self.repository_vm?
+    when 'RedHat'
+      return [] if storage_id.blank?
+    when 'Microsoft'
```

Comment:
This is an oops as well because I think I added back the line for RedHat during the merge that was there the last time but modified when redhat/infra_manager/vm_or_template_shared/scanning.rb was updated.  I'll remove the Redhat and Microsoft lines here and add the correct code in the Microsoft directory.